docs/pages/components/ProgressBar.mdx

```diff
@@ -13,6 +13,14 @@ import { ProgressBar } from '@deque/cauldron-react'
 
 ## Examples
 
+```jsx example
```

Comment:
With multiple examples here, we probably want each example under a level 3 heading.